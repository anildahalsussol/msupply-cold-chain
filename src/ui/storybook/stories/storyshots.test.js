// import path from 'path';
// import initStoryshots from '@storybook/addon-storyshots';

// beforeEach(() => jest.useFakeTimers());

it('', () => {
  expect(true).toBe(true);
});

// initStoryshots({
//   configPath: path.resolve(__dirname, '../'),
// });
