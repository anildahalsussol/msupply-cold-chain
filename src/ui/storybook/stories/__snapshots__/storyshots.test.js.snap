// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots BoldText basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Bold",
      "fontSize": 18,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots BoldText long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Bold",
      "fontSize": 18,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots BoldText with white 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Bold",
        "fontSize": 18,
      }
    }
  >
    3.2c
  </Text>
</View>
`;

exports[`Storyshots Button basic 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "marginTop": 50,
      "width": "50%",
    }
  }
>
  <RNGestureHandlerButton
    activeOpacity={0.3}
    borderless={true}
    collapsable={false}
    onActiveStateChange={[Function]}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        Object {
          "opacity": 1,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "rgba(255,255,255,0.4)",
          "borderRadius": 37.5,
          "borderWidth": 2,
          "height": 60,
          "justifyContent": "center",
          "width": 200,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgba(255,255,255,0.4)",
            "fontFamily": "Roboto-Bold",
            "fontSize": 18,
          }
        }
      >
        Press Me
      </Text>
    </View>
  </RNGestureHandlerButton>
</View>
`;

exports[`Storyshots Divider basic 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#313B4C",
      "height": 1,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Header basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Bold",
      "fontSize": 70,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots Header long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Bold",
      "fontSize": 70,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots Header temperature 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Bold",
      "fontSize": 70,
    }
  }
>
  3.2c
</Text>
`;

exports[`Storyshots LargeText basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 40,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots LargeText long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 40,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots LargeText with white 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 40,
      }
    }
  >
    3.2c
  </Text>
</View>
`;

exports[`Storyshots MediumText basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 26,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots MediumText long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 26,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots MediumText with white 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 26,
      }
    }
  >
    3.2c
  </Text>
</View>
`;

exports[`Storyshots NormalWhite basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 18,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots NormalWhite long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 18,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots NormalWhite with white 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 18,
      }
    }
  >
    3.2c
  </Text>
</View>
`;

exports[`Storyshots SmallText basic 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 12,
    }
  }
>
  Hello Storybook
</Text>
`;

exports[`Storyshots SmallText long text 1`] = `
<Text
  style={
    Object {
      "color": "#22C7FC",
      "fontFamily": "Roboto-Regular",
      "fontSize": 12,
    }
  }
>
  Hello Storybookkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
</Text>
`;

exports[`Storyshots SmallText with white 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 12,
      }
    }
  >
    3.2c
  </Text>
</View>
`;
